spec/harvesting/models/time_entry_spec.rb

```diff
@@ -82,25 +82,30 @@
   describe "#update" do
     let(:attrs) do
       {
-          project: {
-              id: project_id
+          "project" => {
+              "id" => project_id.to_s
           },
-          task: {
-              id: task_id
+          "task" => {
+              "id" => task_id.to_s
           },
-          spent_date: date,
-          hours: '1.0',
-          user: {
-              id: user_id
+          "spent_date" => date,
+          "hours" => '1.0',
+          "user" => {
+              "id" => user_id.to_s
           },
-          is_running: 'false',
-          notes: 'hacked the things'
+          "is_running" => 'false',
+          "notes" => 'hacked the things'
       }
     end
 
     context "when updating an existing time entry" do
-      xit "updates the amount of hours" do
-        # TODO: fixme
+      it "updates the amount of hours" do
+        time_entry.save
+        time_entry.hours = '4.0'
+        time_entry.save
+
+        final_time_entry = Harvesting::Models::TimeEntry.get(time_entry.id)
+        expect(final_time_entry.hours.to_s).to eq('4.0')
```

Comment:
@mscottford I think a more real scenario would be: 

```
time_entry.hours = 4.0
time_entry.save

final_time_entry = Harvesting::Models::TimeEntry.get(time_entry.id)
expect(final_time_entry.hours.to_s).to eq(4.0)
```

You don't need to use strings for setting or comparing... it could just be a float.

Reply:
I'll take a peek at that. I think the value is coming through the JSON back from Harvest as a string, so we'll just need to add some logic to coerce the two different types. Perhaps [Coercible](https://github.com/solnic/coercible).

Reply:
Odd... I was able to leave the comparison as just a float. Perhaps I was extra tired when I originally submitted this.

Reply:
@mscottford I'm not sure, but I think `JSON.parse` is smart enough to cast the object to the right type. :)